tensorflow/python/framework/sparse_tensor.py

```diff
@@ -417,3 +420,62 @@ def is_sparse(x):
     `True` iff `x` is a `tf.SparseTensor` or `tf.compat.v1.SparseTensorValue`.
   """
   return isinstance(x, (SparseTensor, SparseTensorValue))
+
+def from_dense(dense, name=None):
+    """Convert a dense `np.array` or a dense tensor to a `tf.SparseTensor`
+
+    The code to convert a dense tensor to a `SparseTensor` is borrowed from
```

Comment:
This should not be in the docstring

Reply:
Hmm, it is not clear to me what should not be included in the docstring. Can you please elaborate?

Reply:
I meant the paragraph about borrowed code. Users don't care.